multi-agent/lib/mas/elements/nodes/deep_agent/openshell_backend.py

```
@@ -0,0 +1,126 @@
+        """
+        try:
+            result = self._sandbox.exec(
+                ["bash"],
+                stdin=command.encode("utf-8"),
+                timeout_seconds=timeout,
+            )
+            output = result.stdout
+            if result.stderr:
+                output += "\n" + result.stderr
+            return ExecuteResponse(output=output, exit_code=result.exit_code)
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)